### ISSUE TITLE
Error on run "npm i node-red-contrib-nbrowser"

### ISSUE THREAD

**Issue by chameleonbr** [Steveorevo/node-red-contrib-nbrowser#8]
I've tried with and without --unsafe-perm option and received same error.

```npm ERR! Error while executing:
npm ERR! /usr/bin/git ls-remote -h -t git://github.com/segmentio/nightmare.git
npm ERR! 
npm ERR! fatal: Cannot change to '/root/.node-red/..': Permission denied
npm ERR! 
npm ERR! exited with error code: 128

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2018-02-01T13_14_52_816Z-debug.log
```

**Comment by chameleonbr** [comment]
If I change the dependencies all works ok.

```
"dependencies": {
    "nightmare": "^2.10.0",
    "nightmare-download-manager": "^0.2.4",
    "nightmare-upload": "^0.1.1",
    "unfluff": "^3.0.0"
  },
```

**Comment by Steveorevo** [comment]
Interesting. We'll need further steps to reproduce the issue; OS platform/version?

**Comment by chameleonbr** [comment]
I'm behind HTTP proxy, git:// not working with HTTP proxy.
It is also common practice for the dependencies to be bound by the npm itself and versioned. So if someone changes the working of a library does not break your package.
Look at NodeRED package.json (https://github.com/node-red/node-red/blob/master/package.json) as example.

**Comment by Steveorevo** [comment]
Ahh, yes I see a few packages that could conflict without a forward compatible identifier ^. Will make the change and test further.

**Comment by Steveorevo** [comment]
Resolved in 9463b885534358e29ce304c741cf6f568521c26f. New version 1.1.2 pushed. flows.nodered.org should update shortly.

**Comment by chameleonbr** [comment]
Thanks for fix. It's working now.
